guides/README.md

```diff
@@ -22,4 +30,24 @@ This is a collection of guides maintained by the Content team.
 - Relax and unwind
 - Thank you for running the meeting!
 
+## Communication how and when to do it
+
+### Use of hashtags
+
+Hashtags are primarily used in social networks, in particular Twitter, Facebook, Mastodon, etc. There have been occasions where they have been used to identify a campaign in emails or discussion in IRC e.g. #RustReach and #Rust2018. 
+
+For the purpose of providing a tangible example for this guide, we'll use Twitter are our point of reference and it's verb "Tweet" as the way to write an update on the social network.
+
+It is important that such tags are not misappropriated by Rust related items that are not on topic. Just as you would not send a tweet with #RustLang hashtag if it was not Rust related, you should not misuse the following hashtags:
+
+`#RustReach` - This is a specific Rust team programme to grow the community's project collaborators and leaders by reaching to people whose groups and backgrounds are underrepresented in the Rust community and the technology world as a whole. This is not a general drive to extend the "reach" of Rust to the community as a whole.
+
+Whilst we don't have a definitive list, use your judgement and if in doubt ask before sending a tweet with the hashtag.
+
+### Spamming
```

Comment:
I think "repeat messaging" merits its own chapter. Why you are doing it and when it makes sense and how to do it properly to not make it feel like spamming.